person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @id = 0
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    return true if @age >= 18 || @parent_permission
```

Comment:
- [ ] Since you have created of_age?() method, I would implore you to modify this to `return true if of_age?() || @parent_permission` which does the same thing as what you wrote but makes use of the privatized method.